### ISSUE TITLE
As Eva I want to receive links to parts of the app which shows new features

### ISSUE THREAD

**Issue by mkristian** [buzzn/scrum#56]
PR: 

---
**Story:**
As Eva or PhO  I want to receive links to parts of the app which shows new features. Currently these links are only valid until the next deploy. The fix for this would be to switch from UUID to regular integer IDs in the DB, i.e. during beekeeper import the IDs will remain the same and after a staging deploy this will remain as they are. This also makes a few of  our test more robust as things will have a natural order in the DB and on debugging the UI it helps (at least me) to find object in the response json.

---
**XD**

---
**Definition of done**

excluded: 

---

**Technical details**

**Comment by phillipoertel** [comment]
@mkristian Nice one :-) I agree having stable identifiers for our localpools, oranizations etc. would be valuable. Could you rephrase the story as one sentence, written from an end user's perspective? Here's a template to help with that (taken from https://www.mountaingoatsoftware.com/agile/user-stories):

```
As {type of user}
I want {some goal}
so that {some reason}
```

With this clear statement of the requirement and value (who, what, why), @evaklopp can then get an estimate on the effort. These two things (value & effort) then determine the priority in the backlog.

**Comment by evaklopp** [comment]
@mkristian @phillipoertel please help with the criteria for the DoDs, I can´t define it without your help. Thanks.

**Comment by phillipoertel** [comment]
@evaklopp perfect for an estimation meeting, so please introduce this story whenever you find it useful. There's conceptual overlap with the "expressive page titles" story (https://github.com/buzzn/scrum/issues/57), It might make sense to address them in one session.
